KS.Fiks.Maskinporten.Client/MaskinportenToken.cs

```diff
@@ -9,11 +9,13 @@ public class MaskinportenToken
     {
         private readonly DateTime _requestNewTokenAfterTime;
 
+        public int ExpiresIn { get; }
```

Comment:
Ikke så veldig lett å vurdere impact når det ikke er noe å lese. Noen spesiell grunn til å gjøre dette i en egen PR?

Reply:
Ser den. Trodde jeg hadde klar den andre PR'en som bruker dette i Fiks-IO-Client for .NET, men det var noe der som skurret. Jeg håper å ha det klart snart.

Reply:
Her er den andre pull-requesten som trenger dette: https://github.com/ks-no/fiks-io-client-dotnet/pull/180

Reply:
Hadde det ikke vært lettere å gjort alt i en PR?

Reply:
Du har ikke noe test eller noe som bruker det nye feltet?